layers/+email/notmuch/packages.el

```diff
@@ -9,7 +9,14 @@
 ;;
 ;;; License: GPLv3
 
-(setq notmuch-packages '(notmuch helm-notmuch))
+(setq notmuch-packages '(notmuch
+                         (helm-notmuch :toggle (configuration-layer/layer-usedp 'helm))
+                         (counsel-notmuch
+                          :toggle (configuration-layer/layer-usedp 'ivy)
+                          :location (recipe
+                                     :fetcher github
+                                     :repo "fuxialexander/counsel-notmuch")))
+      )
```

Comment:
Don’t leave Parens on their own lines

Reply:
@FrancescElies FYI: `counsel-notmuch` is now on MELPA. You can install it without QUELPA now. Thanks for your PR!